### ISSUE TITLE
Error: Not in GZIP format

### ISSUE THREAD

**Issue by matentzn** [ontodev/robot#536]
```
robot -vvv merge -I http://purl.obolibrary.org/obo/chebi.owl.gz -o chebitest.owl
```

results in 

```
2019-07-16 13:53:16,659 DEBUG org.obolibrary.robot.ExceptionHelper - Missing exception ID: Not in GZIP format
Not in GZIP format
java.util.zip.ZipException: Not in GZIP format
	at java.util.zip.GZIPInputStream.readHeader(GZIPInputStream.java:165)
	at java.util.zip.GZIPInputStream.<init>(GZIPInputStream.java:79)
	at java.util.zip.GZIPInputStream.<init>(GZIPInputStream.java:91)
	at org.obolibrary.robot.IOHelper.loadCompressedOntology(IOHelper.java:1193)
	at org.obolibrary.robot.IOHelper.loadOntology(IOHelper.java:437)
```

Is this a bug in the GZIP input reader?

I recently saw that chebi offered a gzipped version as well, so I would be really keen to be able to read it with ROBOT (1.4.1)!

**Comment by rctauber** [comment]
It looks like it's a problem loading GZIP with the `--input-iri` option, since this works fine:

```
curl -Lk -o chebi.owl.gz http://purl.obolibrary.org/obo/chebi.owl.gz \
&& robot merge -i chebi.owl.gz -o chebitest.owl
```

We need to follow the redirects in a URL manually for the input stream. See PR #537

**Comment by rctauber** [comment]
Resolved by #537 - please reopen if this needs more discussion.
